apps/zbugs/vite.config.ts

```diff
@@ -5,7 +5,7 @@ import tsconfigPaths from 'vite-tsconfig-paths';
 import {makeDefine} from '../../packages/shared/src/build.ts';
 
 async function configureServer(server: ViteDevServer) {
-  const {fastify} = await import('./api/index.js');
+  const {fastify} = await server.ssrLoadModule('/api/index.ts');
```

Comment:
This change broke it for me :-)

With ssrLoadModule we end up loading another copy of the code and the symbols no longer match.